app/models/user.js

```diff
@@ -36,13 +39,28 @@ module.exports = (sequelize, DataTypes) => {
         allowNull: false,
         type: DataTypes.ENUM('ADMIN', 'REGULAR'),
         defaultValue: 'REGULAR'
+      },
+      position: {
+        allowNull: false,
+        type: DataTypes.ENUM,
+        values: USER_POSITIONS,
```

Comment:
Since this would be an object, you can do it like this
```suggestion
        values: Object.values(USER_POSITIONS),
```